validation/input/input_1/input_script_1.py

```diff
@@ -29,9 +29,20 @@ def test_progress():
         time.sleep(0.001)
 
 
-def test_cwd():
-    assert (Path(os.environ["INPUT_FOLDER"]) / "input_script_1.py").exists()
-    assert (Path(os.environ["INPUT_FOLDER"]) / "requirements.txt").exists()
+def test_inputs():
+    assert (Path(os.environ["INPUT_1"]) / "input_script_1.py").exists()
+    assert (Path(os.environ["INPUT_1"]) / "requirements.txt").exists()
+
+
+def test_outputs():
+    """write a file in each output"""
+    for output_folder_env in [*(f"OUTPUT_{i}" for i in range(1, 5))]:
```

Comment:
MINOR: no need to expand in list, can use directly generator to iterate ...
```python
   for output_folder_env in ( f"OUTPUT_{i}" for i in range(1, 5) ) :
      ... 
```

Reply:
correct... bad copy-pasting...